test/com/google/enterprise/adaptor/opentext/OpentextAdaptorTest.java

```diff
@@ -885,7 +885,19 @@ public void testDoContainer() throws IOException {
   }
 
   @Test
-  public void testGetDisplayUrl() {
+  public void testInvalidDisplayUrl() throws Exception {
```

Comment:
I think we should include tests that an invalid queryString (and an invalid objectAction) are also caught.